lib/usd/ui/cmds/USDImportDialogCmd.cpp

```diff
@@ -14,9 +14,9 @@
 // limitations under the License.
 //
 
-#include "USDImportDialogCmd.h"
-#include "views/USDImportDialog.h"
-#include "views/USDQtUtil.h"
+#include <mayaUsdUI/ui/cmds/USDImportDialogCmd.h>
```

Comment:
I've noticed many places where you've done this. 

According to the coding guidelines doc the related header should be first and just "" with no paths.
https://github.com/Autodesk/maya-usd/blob/dev/doc/codingGuidelines.md#include-order